src/Microsoft.Health.Fhir.Shared.Api/Modules/MediationModule.cs

```diff
@@ -25,12 +25,6 @@ public void Load(IServiceCollection services)
 
             services.AddMediatR(KnownAssemblies.All);
 
-            Predicate<Type> isPipelineBehavior = y => y.IsGenericType && y.GetGenericTypeDefinition() == typeof(IPipelineBehavior<,>);
-
-            services.TypesInSameAssembly(KnownAssemblies.All)
-                .Transient()
-                .AsImplementedInterfaces(isPipelineBehavior);
-
```

Comment:
What did this do? It doesn't look like it fits with the rest of the code review.